config.ts

```diff
@@ -137,3 +137,5 @@ module.exports = {
       minorVersion: null
     }
 };
+
+export default {};
```

Comment:
Is this necessary? Style guide forbids default exports.

Here and elsewhere.

Reply:
It's not necessary in this file specifically, but I've found that omitting any sort of export in a file causes the objects declared in that file to be dumped in the global scope rather than module scope upon being imported through `import`. Adding this line is a workaround to prevent that; in a future PR, all of these lines will be removed.